app/cdap/components/RecipeDetails/DetailContainer/index.tsx

```diff
@@ -0,0 +1,175 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Box, Typography } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+import T from 'i18n-react';
+import { dateFormatting } from 'components/RecipeDetails/utils';
+
+interface IRecipeId {
+  namespace: {
+    name: string;
+    generation: number;
+  };
+  recipeId: string;
+}
+
+interface IRecipeItem {
+  recipeId: IRecipeId;
+  recipeName: string;
+  description: string;
+  directives: string[];
+  createdTimeMillis: number;
+  updatedTimeMillis: number;
+  recipeStepsCount: number;
+}
+
+interface IRecipeDetailsProps {
+  recipeDetails: IRecipeItem;
+}
+
+const PREFIX = 'features.WranglerNewUI.RecipeDetails';
+
+const RecipeDetailWrapper = styled(Box)`
+  margin-top: 19px;
+`;
+
+const RecipeName = styled(Typography)`
```

Comment:
please rename this to label
as we have this same label in multiple other features, we may reuse this from some other place going forward

Reply:
Done